.github/workflows/clean-pr.yaml

```diff
@@ -47,11 +53,11 @@ jobs:
         run: |
           docker_registry="europe-west3-docker.pkg.dev/${{ secrets.GOOGLE_HOPRASSOCIATION_PROJECT }}/docker-images"
           next_version=$(./scripts/get-next-version.sh Build ${{ github.event.inputs.pr_number }})
-          ./scripts/bump-version.sh ${next_version}
-          images=('hopli' 'hoprd' 'hopr-anvil' 'hopr-pluto' 'hopr-toolchain' 'hoprd')
+          images=('hopli' 'hoprd' 'hopr-anvil' 'hopr-pluto' 'hopr-toolchain')
           for image in "${images[@]}"; do
-            gcloud artifacts docker images delete --quiet --delete-tags --async ${docker_registry}/$image:2.0.0-rc.5-pr.5496
-            gcloud artifacts docker images delete --quiet --delete-tags --async ${docker_registry}/$image:2.0.0-rc.5-pr.5496-cache
+            echo "Removing tag: $image:${next_version} and $image:${next_version}-cache"
+            gcloud artifacts docker images delete --quiet --delete-tags --async ${docker_registry}/$image:${next_version} || true
+            gcloud artifacts docker images delete --quiet --delete-tags --async ${docker_registry}/$image:${next_version}-cache || true
```

Comment:
It would be better, if this existed e.g. as part of a script or a separate mini script called by the CI, e.g.

`remove_image_from_registry.sh <image_name>`

So that it can also be easily invoked manually if needed.

Reply:
It won't be that easy, as the developer needs to have installed: gcloud, gh
And the script not only removes images, but also removes gh cache, and nix cache
I will consider as an improvement for next sprints.